cmd/juju/commands/cmd_test.go

```diff
@@ -64,7 +62,6 @@ func TestRunMain(t *stdtesting.T) {
 func badrun(c *gc.C, exit int, args ...string) string {
 	localArgs := append([]string{"-test.run", "TestRunMain", "-run-main", "--", "juju"}, args...)
 	ps := exec.Command(os.Args[0], localArgs...)
-	ps.Env = append(os.Environ(), osenv.JujuXDGDataHomeEnvKey+"="+osenv.JujuXDGDataHome())
```

Comment:
No - the environment variable is already set, so appending the value
had no effect previously, but the new Go changes (and the new Setenv
primitive) assumes that it does.